docs/design/2019-11-14-tidb-builtin-diagnostics-zh_CN.md

```diff
@@ -289,7 +289,7 @@ message ServerInfoResponse {
 
 - 终端用户：用户直接通过 SQL 查询获取集群信息排查问题
 - 运维系统：TiDB 的使用环境比较多样，客户可以通过 SQL 获取集群信息将 TiDB 集成到自己的运维系统中
-- 生态工具：外部工具通过 SQL 拿到集群信息实现功能定制，比如 `[sqltop](https://github.com/ngaut/sqltop)` 可以直接通过集群 `events_statements_summary_by_digest` 获取整个集群的 SQL 采样信息
+- 生态工具：外部工具通过 SQL 拿到集群信息实现功能定制，比如 `[sqltop](https://github.com/ngaut/sqltop)` 可以直接通过集群 `statements_summary` 获取整个集群的 SQL 采样信息
```

Comment:
this no longer stands

Reply:
Why? It seems this tool can access new `statements_summary` table. Do you mean we can access `cluster_statements_summary` directly?